ssc_interface_wrapper_ros2/src/converter_worker.cpp

```diff
@@ -282,7 +285,7 @@ namespace ssc_interface_wrapper{
                                   const automotive_platform_msgs::msg::CurvatureFeedback& msg_curvature,
                                   const automotive_platform_msgs::msg::SteeringFeedback& msg_steering_wheel)
     {
-        if(velocity_msg_exists_ && curvature_msg_exists_ && steering_msg_exists_ && (received_new_msg_  || single_update_processed_))
+        if(velocity_msg_exists_ && curvature_msg_exists_ && steering_msg_exists_)
```

Comment:
Maybe you should pass these flags into the function so that both functions can be called in a row without side affecting. Same with the other sync function